src/dex/balancer-v1/config.ts

```diff
@@ -0,0 +1,57 @@
+import { DexParams } from './types';
+import { DexConfigMap } from '../../types';
+import { Network } from '../../constants';
+import { SwapSide } from 'paraswap-core';
+
+// These are required to filter out log calls from the event calls
+export const LogCallTopics = [
+  '0xb02f0b7300000000000000000000000000000000000000000000000000000000',
+  '0x5db3427700000000000000000000000000000000000000000000000000000000',
+  '0x46ab38f100000000000000000000000000000000000000000000000000000000',
+  '0x4f69c0d400000000000000000000000000000000000000000000000000000000',
+  '0x8201aa3f00000000000000000000000000000000000000000000000000000000',
+  '0x7c5e9ea400000000000000000000000000000000000000000000000000000000',
+  '0x34e1990700000000000000000000000000000000000000000000000000000000',
+  '0x49b5955200000000000000000000000000000000000000000000000000000000',
+  '0x4bb278f300000000000000000000000000000000000000000000000000000000',
+  '0x3fdddaa200000000000000000000000000000000000000000000000000000000',
+  '0xe4e1e53800000000000000000000000000000000000000000000000000000000',
+  '0xcf5e7bd300000000000000000000000000000000000000000000000000000000',
+  '0x02c9674800000000000000000000000000000000000000000000000000000000',
+];
+
+export const BALANCER_CHUNKS = 10;
```

Comment:
It is event base implementation. I think we can remove CHUNKING related logic

Reply:
done b39a20849546ea7748e3fd8659e2545af1b05796